roles/inspec/templates/etc/inspec/host-controls/attributes.yml

```diff
@@ -1,3 +1,6 @@
 {% for name,value in inspec.openstack.horizon.attributes.iteritems() %}
 {{ name }}: {{ value }}
 {% endfor %}
+{% for name,value in inspec.rhel7.attributes.iteritems() %}
```

Comment:
agreed!   although better might be a check for the OS and only add it in for rhel7 inside the template itself.